Cloud Changelog - June 30th (#4668)
mattermost/docs: source/cloud/cloud-administration/cloud-changelog.md

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

## Release 2021-06-30

### Highlights

> 

Reviewer: Collapsed Reply Threads (Early Beta Access)

We're excited to give you early access to Collapsed Reply Threads (Beta). It can be enabled in the System Console > Experimental > Collapsed Reply Threads (Beta)

Author: @esethna Based on [Confluence](https://mattermost.atlassian.net/wiki/spaces/Threads/pages/1972469761/Cloud+and+Self-Managed+Release+Plan), I thought we ship marketing for Cloud July 14th release?

Reviewer: Ah, sorry to clarify: By Marketing I'm referring to blog posts, tweets, etc. I think it's fine to note in the changelog for cloud @amyblais

#### Emoji Enhancements with Skin Tone Selection

>  - Added support for emoji standard v13.0. Users will see Reactions and Emoji rendered based on their operating system or device, and have the ability to choose various skin tones using the Mattermost emoji picker.

Reviewer: > Emoji rendered based on their operating system or device

I don't know if this is true? Asked here: https://community-daily.mattermost.com/private-core/pl/gk4b7i5esfn3bxgzept7xe79ye

Author: Removed that part.

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

## Release 2021-06-30

### Highlights

> ### Unlimited E10/20 trials

Reviewer 2: ```suggestion
### Unlimited E10/20 trials prohibited
```

Reviewer 3: 1. I would remove from Cloud changelog, since it only affects Self-Managed (Cloud has no E10/E20 trials).
2. For self-managed changelog on 7/16 release, I would adjust this to say

```
#### Improved Enterprise trial experience

- After a Self-Managed trial ends, admins can optionally contact sales or make a purchase in a single click.
```

 - Users are now prohibited from starting unlimited E10/20 trials.

### Emoji Enhancements with Skin Tone Selection
 - Added support for emoji standard v13.0. Users will see Reactions and Emoji rendered based on their operating system or device, and have the ability to choose various skin tones using the Mattermost emoji picker.

### Improvements

#### User Interface (UI)

>  - Added English-Australian English language variation.

Reviewer 2: ```suggestion
 - Added English-Australian language variation.
```

>  - Added the ability to upload ``.jpeg`` files on Linux.

Reviewer 2: question: are both .jpeg and .jpg files supported? If so, consider adding this level of detail to the changelog

Author: .jpeg files were already supported, I added a note.

>  - Before anything is typed, a list of channels you are a member of are now shown and sorted by recency. The number of channels shown here in the default state are capped to maximum of 20. If there is no recent channel activity (in the case of new users), an alphabetical list of channels is shown showing **My Channels** first and other **Public Channels** next, still capped at 20 for the default state.

Reviewer 2: ```suggestion
 - Before anything is typed, a list of channels you are a member of is now shown and sorted by recency. The number of channels shown here in the default state are capped to maximum of 20. If there is no recent channel activity (in the case of new users), an alphabetical list of channels is shown displaying **My Channels** first and other **Public Channels** next, capped at a maximum of 20 by default.
```

Reviewer 2: Can we clarify where in the product UI users will see this update? It's not clear on first read.

>  - Polish, German, and Italian languages were downgraded to beta.

Reviewer 2: Does this statement convey everything we need to communicate about this? What do users need to know about a given supported language that's downgraded to beta?

Author: Added a note and a link to docs.

#### Administration

>  - Optimized the bulk import process by removing the need for the server to write the incoming archive to the filesystem when unzipping it.

Reviewer 2: ```suggestion
 - Optimized the bulk import process by no longer requiring the server to write the incoming archive to the filesystem when unzipping it.
```

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

## Release 2021-06-30

### Highlights

#### Collapsed Reply Threads (Early Beta Access)

>  - We're excited to give you early access to Collapsed Reply Threads (Beta). It can be enabled in the **System Console > Experimental > Collapsed Reply Threads (Beta)**. Learn more about the features and known issues in [our documentation](https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta).

Reviewer: Wrong link. https://docs.mattermost.com/help/messaging/organizing-conversations.html
